corehq/apps/app_manager/urls.py

```diff
@@ -70,7 +70,7 @@
 urlpatterns = [
     url(r'^$', view_app, name='default_app'),
     url(r'^xform/(?P<form_unique_id>[\w-]+)/$', xform_display, name='xform_display'),
-    url(r'^browse/(?P<app_id>[\w-]+)/modules-(?P<module_id>[\w-]+)/forms-(?P<form_id>[\w-]+)/source/$',
+    url(r'^browse/(?P<app_id>[\w-]+)/(?P<form_unique_id>[\w-]+)/source/$',
```

Comment:
They'll get an error if they have form view open before deploy and then after deploy try to either download the form or view it in a popup; it doesn't break the whole page. But you're right that there's no reason not to support both.

https://github.com/dimagi/commcare-hq/pull/16208/commits/956d76675a5afd1464acbccb254de88e75aa9021